### ISSUE TITLE
Search Extensions: Quicksearch URL launch

### ISSUE THREAD

**Issue by dwertheimer** [NotePlan/plugins#488]
I don't know if you consider this a bug or not, but I would have thought that if I ran a vanilla X-Callback to call a command, it would have had the same result as typing that command in the commandbar. however, when I click the URL:
```
[Link](noteplan://x-callback-url/runPlugin?pluginID=jgclark.SearchExtensions&command=quickSearch)
```
I get this: 
<img width="258" alt="Screen Cap 2023-08-30 at 11 01 28@2x" src="https://github.com/NotePlan/plugins/assets/8949588/07c0571c-1223-4277-990e-31c6dfbda4e8">

```
Executing function 'quickSearch'
2023-08-30 11:01:26 🥺 WARN 🥺 normaliseSearchTerms :: No valid search terms found in []
```

If there's another way to launch QuickSearch with the dialog box, pls let me know. This used to work.

**Comment by dwertheimer** [comment]
@jgclark Is there any workaround that you know of for this issue? I really want to be able to launch quickSearch via xcallback

**Comment by jgclark** [comment]
You mention the search "dialog box". If that's the one with the clever
dialog, that's the /flexiSearch command which does work via:
noteplan://x-callback-url/runPlugin?pluginID=jgclark.SearchExtensions&command=flexiSearch

If you do mean /quickSearch, then the documentation says:
noteplan://x-callback-url/runPlugin?pluginID=jgclark.SearchExtensions&command=quickSearch&
+
search term(s) separated by commas.  That's still the case.

**Comment by dwertheimer** [comment]
Yes, @jgclark  I did mean quickSearch. It's the only one I use on a daily basis. I rarely want to save the results or do anything fancy. I just want an xcallback to bring up the simple quicksearch input box. But when I try to call it with an xcallback, I get the "empty search" error in the OP.

<img width="250" alt="Screen Cap 2023-09-30 at 15 17 43@2x" src="https://github.com/NotePlan/plugins/assets/8949588/c3a7aa23-1935-429a-b4d4-4bdf72ab1b23">

I do love flexiSearch, and so yes, I can use an xcallback for that.
And technically, your README says:
"There is no x-callback equivalent of /flexiSearch, as this is simply a wrapper around these other commands."

As an aside, I would love to use flexisearch on the phone. however, it is hard to see it. you may want to play around with the format so it works equally well on both.

**Comment by jgclark** [comment]
Released v1.2.3 which allows you to run `noteplan://x-callback-url/runPlugin?pluginID=jgclark.SearchExtensions&command=quickSearch` without any parameters, but still get prompted for search terms to use.

**Comment by dwertheimer** [comment]
Thank u @jgclark
